.github/workflows/bot.yml

```diff
@@ -302,15 +301,9 @@ jobs:
           - flinkProfile: 'flink1.14'
             sparkProfile: 'spark3.2'
             sparkRuntime: 'spark3.2.3'
-          - flinkProfile: 'flink1.13'
```

Comment:
We have flink1.14.6 and spark 3.1.3 doker image now, can just upgrade the flink to flink1.14.

Reply:
Done